### PR TITLE
Doc out of stock

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?
New doc for the `OutOfStock` component.

### CODE REVIEW

File: apps/docs/docs/reference/ui/organisms/OutOfStock.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Best practices
+
+### Do's
+- Use the component in a PDP. 
+- Inform the user when their email is incorrect, e.g. *Please enter a valid email address*.
+
+
+### Don'ts
+- Don't use the `OutOfStock` for alert messages, instead use the [`Alert` component](https://www.faststore.dev/reference/ui/molecules/Alert).
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/organisms/OutOfStock.mdx:70]:
```suggestion
- Don't use the `OutOfStock` for alert messages, instead use the [Alert component](https://www.faststore.dev/reference/ui/molecules/Alert).
```

I think we leave it without backticks to follow the others components doc pattern.

File: apps/docs/docs/reference/ui/organisms/OutOfStock.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+`data-out-of-stock-message`
+
+
+## Best practices
+
+### Do's
+- Use the component in a PDP. 
+- Inform the user when their email is incorrect, e.g. *Please enter a valid email address*.
+
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/organisms/OutOfStock.mdx:67]: I think we can also add something like this:
- It should be placed near the product and/or clearly mention the product that the user is subscribing to get notified when that item returns to stock

**Review by hellofanny — APPROVED** [review]
Niceeee mariii 🤩 I've left a few suggestions. Can you check if it makes sense? 🙈

File: apps/docs/docs/reference/ui/organisms/OutOfStock.mdx (hunk 2 above)
Comment (Mariana-Caetano) [reply apps/docs/docs/reference/ui/organisms/OutOfStock.mdx:67]: Good suggestion, Fanny!

**Review by carolinamenezes — APPROVED** [review]

File: apps/docs/docs/reference/ui/organisms/OutOfStock.mdx [hunk 3]
```diff
@@ -58,17 +50,16 @@ import {
...
 
-
 ## Best practices
 
 ### Do's
-- Use the component in a PDP. 
-- Inform the user when their email is incorrect, e.g. *Please enter a valid email address*.
-- Place the component near the product and/or indicate to subscribed users that they will be notified when the item returns to stock.
+- Use the `OutOfStock` component in PDPs.
+- Specify the input type in your form to validate data (e.g., ` <Input type="email" placeholder="Email"/>`).
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/organisms/OutOfStock.mdx:57]: Niceee!

**Pull request merged**